cg/store/models.py

```diff
@@ -758,3 +758,18 @@ def to_dict(self) -> dict:
 
     def __str__(self) -> str:
         return self.name
+
+
+class SequencingStats(Model):
```

Comment:
Some comments above on the naming of the fields in the SequencingStats model, maybe we can name it SequencingStatistics instead.